### PR TITLE
Chillx Fix

### PR DESCRIPTION
**Pull request by phisher98**

### CODE REVIEW

**Review by fire-light42 — CHANGES_REQUESTED** [review]
Dont use android libs in extractors, as the core of cloudstream should be cross platform. If I remember correctly importing android will break the auto build.

**Comment by phisher98** [comment]
@fire-light42 I have removed 
import android.annotation.TargetApi
import android.os.Build

still giving error

**Comment by phisher98** [comment]
@fire-light42 fixed

File: library/src/commonMain/kotlin/com/lagradost/cloudstream3/extractors/Chillx.kt [hunk 1]
```diff
@@ -132,45 +111,55 @@ open class Chillx : ExtractorApi() {
...
+            val keyValue = xorKey[i % xorKey.size]  
+            val xorResult = byteValue xor keyValue
+            result.append(xorResult.toChar())
         }
+
+        return result.toString()
+    }
+
+    private suspend fun fetchKey(): String? {
+        return app.get("https://raw.githubusercontent.com/Rowdy-Avocado/multi-keys/refs/heads/keys/index.html")
```
Comment (fire-light42) [inline library/src/commonMain/kotlin/com/lagradost/cloudstream3/extractors/Chillx.kt:160]: Cache this value

**Review by fire-light42 — COMMENTED** [review]
Looking at this, it looks like there is 3 different locations that get the same url. Can you please unify them in a companion object that all extractors that use these keys share? https://github.com/recloudstream/cloudstream/blob/678ddb6b0a1bc1a05c81ae65bf9d03614ce27f74/library/src/commonMain/kotlin/com/lagradost/cloudstream3/extractors/VidSrcTo.kt#L25

**Comment by phisher98** [comment]
@fire-light42 Why its failing its working in local

**Comment by phisher98** [comment]
@fire-light42 Hope this works now

**Review by fire-light42 — APPROVED** [review]

**Pull request merged**